### PR TITLE
Feature/actuators unit tests

### PR DESCRIPTION
**Pull request by menglingwei**
add machine unit test cases

### CODE REVIEW

File: pkg/actuators/machine/stubs.go
Comment (JoelSpeed) [inline pkg/actuators/machine/stubs.go:80]: What's the reason for making this public? Is it being used in other test suites?

File: pkg/actuators/machine/instances_test.go
Comment (JoelSpeed) [inline pkg/actuators/machine/instances_test.go:895]: Are we not expecting this function to filter out duplicate values?

**Review by alexander-demichev — CHANGES_REQUESTED** [review]

File: pkg/actuators/machine/stubs.go
Comment (menglingwei) [reply pkg/actuators/machine/stubs.go:80]: change to generateAlibabaCloudCredentialsSecretFromEnv

File: pkg/actuators/machine/instances_test.go
Comment (menglingwei) [reply pkg/actuators/machine/instances_test.go:936]: i fixed func to filter duplicate key.

File: pkg/actuators/machine/instances_test.go
Comment (menglingwei) [reply pkg/actuators/machine/instances_test.go:895]: i fixed func to filter duplicate key.

File: pkg/actuators/machine/actuator_test.go
Comment (menglingwei) [reply pkg/actuators/machine/actuator_test.go]: renamed

File: pkg/actuators/machine/filter_test.go
Comment (menglingwei) [reply pkg/actuators/machine/filter_test.go]: renamed

File: pkg/actuators/machine/instances_test.go
Comment (menglingwei) [reply pkg/actuators/machine/instances_test.go]: renamed

File: pkg/actuators/machine/instances_test.go
Comment (menglingwei) [reply pkg/actuators/machine/instances_test.go]: renamed

File: pkg/actuators/machine/instances_test.go
Comment (menglingwei) [reply pkg/actuators/machine/instances_test.go]: renamed

File: pkg/actuators/machine/instances_test.go
Comment (gauravsingh85) [reply pkg/actuators/machine/instances_test.go:1148]: @menglingwei can you reply to Joel comments .

Comment (menglingwei) [reply pkg/actuators/machine/instances_test.go:1148]: Sorry, I missed this comments. I updated the test cases.

**Comment by JoelSpeed** [comment]
I think this is good to go, though I'd like to see if I can find a second pair of eyes to review it as well

File: pkg/actuators/machine/machine_scope_test.go
Comment (alexander-demichev) [inline pkg/actuators/machine/machine_scope_test.go:36]: this package is imported twice

**Pull request merged**
